docs/example-scenario/data/data-contextualization-based-on-azure-sql-graph-content.md

```diff
@@ -70,13 +70,13 @@ The solution assumes that a graph has already been created in a graph database.
 
 The solution uses Azure Databricks for the data contextualization process.
 
-### Graph database
+### Graph data
 
-The graph database is the database that stores the graph models. As noted earlier, there are many graph databases available. In this solution, the [graph capabilities of SQL Server](/sql/relational-databases/graphs/sql-graph-overview?view=sql-server-ver16) are used to create the graph.
+The graph database is the database that stores the graph data/models. As noted earlier, there are many graph databases available. In this solution, we use Azure SQL Database and the [graph capabilities of SQL Server](/sql/relational-databases/graphs/sql-graph-overview?view=sql-server-ver16) are used to create the graph.
```

Comment:
@kenakamu should we say 'we use Azure SQL Database and the [graph capabilities of SQL Server](/sql/relational-databases/graphs/sql-graph-overview?view=sql-server-ver16) to create the graph.'?

Reply:
makes sense